build.gradle

```diff
@@ -1,125 +1,111 @@
-buildscript {
-	repositories {
-		jcenter()
-		maven {
-			name = "Fabric"
-			url = "https://maven.fabricmc.net"
-		}
-	}
-	dependencies {
-		classpath "net.fabricmc:fabric-loom:0.2.0-SNAPSHOT"
-	}
-}
-
 plugins {
-	id "scala"
-	id "idea"
-	id "maven"
-	id "maven-publish"
-	id "com.github.johnrengelman.shadow" version "2.0.4"
-	id 'com.matthewprenger.cursegradle' version "1.1.2"
+    id "java"
+    id "scala"
+    id "maven"
+    id "maven-publish"
+    id 'fabric-loom' version '0.2.2-SNAPSHOT'
```

Comment:
Idealy this should be using 0.2.3 or higher as that has the improved dependencies in the maven pom.